src/handlers/push_message.rs

```diff
@@ -51,6 +51,18 @@ pub async fn handler(
     let client = state.client_store.get_client(&tenant_id, &id).await?;
     info!("fetched client ({}) for tenant ({})", &id, &tenant_id);
 
+    if let Ok(_notification) = state
```

Comment:
The notification may be stored but might now have processed we should have a better way to do this eventually!